client/src/js/components/Accomodations.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const Accomodation = (props) => {
+  console.log();
+  return (
+    <hgroup id="accomodations">
+      <img id="userIcon" alt="" src="https://s3-us-west-1.amazonaws.com/napbnb/icon21.png" />
```

Comment:
are these hard coded src for now or will you have this as template?

Reply:
this will be the template since those icons don't change. Is there abetter way to do this?